[IMP] l10n_it: Improve tax groups & Closing entry

lordkrandel opened the pull request:
Rearrangement of tax groups:
* Make one tax group for tax rate, removing the deductible ones.  `IVA 4% / 5% / 10% / 20% / 21% / 22%`
* Set the Tax current Payable Account `property_tax_payable_account_id` and Tax Current Receivable Account `property_tax_receivable_account_id` to "260500 erario c/IVA"
* Make all taxes with rates = 20% or 21% inactive (these are historical tax rates) 

Task link: https://www.odoo.com/web#id=2577821&model=project.task

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/73374).

william-andre commented on addons/l10n_it/data/account_tax_template.xml:
```diff
@@ -144,6 +144,7 @@
         <field name="sequence">3</field>
         <field name="amount">20</field>
         <field name="amount_type">percent</field>
+        <field name="active">False</field>
```
Making the tax inactive makes sense, but I don't see the point of making the template inactive? There shouldn't be any link to the template.

william-andre reviewed (commented):
This is not greened yet in the pipe, and still needs to go through functional validation

lordkrandel commented:
I have

- renamed `l10n_it_chart_data.xml` (data for chart template insertion) --> `account_chart_template.xml`
- reworked `account.chart.template.csv` (account data for the chart template) --> `account_chart_template_accounts.xml`
- setup the manifest so that it executes this order
   1. `account_chart_template.xml`
   2. `account.account.template.csv`
   3. `account_chart_template_accounts.xml`
- removed the inactive `account.tax.template`s

smetl commented:
@robodoo r+

smetl commented:
@robodoo r-

robodoo commented:
Linked pull request(s) odoo/upgrade#2626 not ready. Linked PRs are not staged until all of them are ready.

smetl commented on addons/l10n_it/data/account_chart_template.xml:
```diff
@@ -0,0 +1,12 @@
+        <field name="code_digits">6</field>
+        <field name="currency_id" ref="base.EUR"/>
+        <field name="country_id" ref="base.ie"/>
+    </record>
```
why changing that? Original values:
```
        <record id="l10n_it_chart_template_generic" model="account.chart.template">
            <field name="name">Italy - Generic Chart of Accounts</field>
            <field name="cash_account_code_prefix">180</field>
            <field name="bank_account_code_prefix">182</field>
            <field name="transfer_account_code_prefix">183</field>
            <field name="currency_id" ref="base.EUR"/>
            <field name="country_id" ref="base.it"/>
        </record>
```
Note: big typo in `<field name="country_id" ref="base.ie"/>`

lordkrandel replied:
Dang! Fixing it NOW.

smetl replied:
:fire: :fire: :ambulance: :hospital: :fire: :fire:

smetl commented:
@robodoo r+
